<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgRegister</class>
 <widget class="QDialog" name="DlgRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TOP</string>
  </property>
  <widget class="QLabel" name="label_regtime">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>授权期限:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_regtime_show">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_regstate">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>授权状态:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_regstate_show">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>未授权</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_reghardwaretime_show">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_reg_hardwaretime">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>硬件时间:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_registercode">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册编码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_regarrow">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_regarrow_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_regarrow_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_syscode">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>系统编码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_systemver_show">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_system_ver">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>版本信息:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_syscode">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_regcode">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_regcode_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_regcode_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_regcode_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_law">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>版权所有：东莞市多普光电设备有限公司</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_law_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>http://www.hk-topopt.com</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_logo">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/logo.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_scancode">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>80</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/scancode.jpg);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_checkcode">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>校验编码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_checknum_show">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>------</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_keyboard">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>611</width>
     <height>213</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
